desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -162,6 +167,13 @@ export class DashCanvasModel extends HoistModel {
     loadState(state) {
         this.clear();
         state.forEach(state => this.addView(state.viewSpecId, state));
+        wait().then(() =>
```

Comment:
Not sure about this, I think we are over-complicating things here - pretty sure all we need is this line after the `this.loadState()` call in the ctor:
```
this.state = this.buildState();
```
That at least resolves my bug about the `state` being undefined for 1 second after creating the DashCanvasModel with initial state (causing some undesired behavior in app logic for checking if the dash state has been modified).